### PR TITLE
pre-placed posters don't retain their pixel offset when taken down carefully

### LINKED ISSUE #13430: Pre-placed posters on maps keep their pixel shift even when taken down with wirecutters and placed elsewhere.
This means you can cut off a poster with wirecutters, and when placing it against a wall it will phase through it into the tile beneath. You can end up with floating posters and so on.

Note that this ONLY happens with posters already placed on the map with an existing pixel shift, spawned posters or unused poster items lying somewhere won't do this.

### LINKED ISSUE #20579: Posters placed can print a square above/below where they were supposed to be placed
An issue that may be exclusive to Meta/Meta Maintenance, but an issue nontheless. Several attempts placing a poster resulted in them being placed above or below the wall, depending on where I was standing in relation to the wall.

### PR DESCRIPTION
**Pull request by Mervill**
:cl: Mervill
fix: pre-placed posters don't retain their pixel offset when taken down carefully
/:cl:

**ATTN Mappers**: you should place posters on wall tiles directly, using the pixel offset method is actually not required in this case.

fixes #13430
fixes #20579

### CODE REVIEW

**Comment by Core0verload** [comment]
Does it really need a changelog?

**Comment by Mervill** [comment]
The changelog is usually for things "visible to players" and this is an art bug ... but yeah it's not the biggest deal in the world

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -225,21 +225,16 @@ list(name = "- Carbon Dioxide", desc = " This informational poster teaches the v
...
-		var/obj/item/weapon/poster/legit/P = new(src, serial_number)
-		P.resulting_poster = src
-		P.loc = location
-		P.pixel_x = 0
-		P.pixel_y = 0
-		loc = P
-
+		P = new /obj/item/weapon/poster/legit(src, serial_number)	
+	P.resulting_poster = src
+	P.loc = location
```
Comment (coiax) [inline code/game/objects/effects/contraband.dm]: P.forceMove(location)

Comment (Mervill) [inline code/game/objects/effects/contraband.dm]: Just for `P.loc = location` right? Github is weird and highlights 3 lines

**Pull request merged**